shared/fs/filepreview/container.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import * as I from 'immutable'
+import {
+  compose,
+  connect,
+  lifecycle,
+  setDisplayName,
+  type Dispatch,
+  type TypedState,
+} from '../../util/container'
+import FilePreview from '.'
+import * as FsGen from '../../actions/fs-gen'
+import * as Types from '../../constants/types/fs'
+import * as Constants from '../../constants/fs'
+
+type OwnProps = {
```

Comment:
shouldn't need to define ownProps or stateprops or dispatch props anymore. lets just remove them

Reply:
Done